IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -137,17 +168,71 @@ def describe_screen(screen_id):
                     "Channels"
                 ]
 
-                # Clean channels value and separate into stain and stain target
-                stain = set()
-                stain_target = set()
-                for entry in channels.split(";"):
-                    temp_list = entry.split(":")
-                    stain.add(temp_list[0])
-                    stain_target.add(temp_list[1])
+                # Clean channels value and add to stain:target list
+                stains_targets = list()
+                for channel in channels.split(";"):
+
+                    # For channel entries with 'stain:target' format
+                    if bool(re.search(r"([\:])+", channel)):
+                        # Split stain and target for whitespace trimming
+                        split = channel.split(":")
```

Comment:
please use a more descriptive variable name (maybe `channel_split`?)

Reply:
@gwaybio This line is redundant so I removed it. I did change `[s.strip() for s in channel.split(:)]` to `[channel_element.strip() for channel_element in channel.split(:)]` to provide a more descriptive name.

Added in a86133dfdab7531fe2e53993e067aa8f93432c24.